scripts/add_translation.py

```diff
@@ -0,0 +1,122 @@
+APPLICATION_NAME = 'International Situations Project'
+
+# Downloaded csv files with each site's consent form
+files_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'consent_forms')
+
+
+def get_credentials():
+    """Gets valid user credentials from storage.
+
+    If nothing has been stored, or if the stored credentials are invalid,
+    the OAuth2 flow is completed to obtain the new credentials.
+
+    Returns:
+        Credentials, the obtained credential.
+    """
+    credential_dir = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'credentials')
+    credential_path = os.path.join(credential_dir, 'credentials.json')
+    store = file.Storage(credential_path)
+    credentials = store.get()
+    if not credentials or credentials.invalid:
+        flow = client.flow_from_clientsecrets(CLIENT_SECRET_FILE, SCOPES)
+        flow.user_agent = APPLICATION_NAME
+        if flags:
+            credentials = tools.run_flow(flow, store, flags)
+        else:  # Needed only for compatibility with Python 2.6
+            credentials = tools.run(flow, store)
+        print('Storing credentials to ' + credential_path)
+    return credentials
+
+
+def main():
+    credentials = get_credentials()
+    http = credentials.authorize(httplib2.Http())
+    service = discovery.build('drive', 'v3', http=http)
+
+    while True:
+        response = raw_input('Enter folder ID\n')
+        # get and download files
+        data = service.files().list(q="'" + response + "'" + " in parents").execute()
+        download_translation_file(data['files'], service)
+        response = raw_input('Add new folder? (y/n)\n')
+        if response == 'y':
+            continue
+        elif response == 'n':
+            print 'Exit..'
+            sys.exit()
+
+def download_consent_files(files, service):
+    for f in files:
+        file_id = f['id']
+        request = service.files().export_media(fileId=file_id, mimeType='text/csv').execute()
+        fn = '%s.csv' % ('consent_forms/' + f['name'].replace(' ', '_'))
+        if request:
+            with open(fn, 'wb') as csvfile:
+                csvfile.write(request)
+
+
+def download_translation_file(files, service):
+    for f in files:
+        file_id= f['id']
+        file_type= f['mimeType']
+        file_name= f['name']
+        if file_type.endswith('.spreadsheet'):
+            request = service.files().export_media(fileId=file_id, mimeType='text/csv').execute()
+            fn = '%s.csv' % (file_name.replace(' ', '_'))
+            if request:
+                with open(fn, 'wb') as csvfile:
+                    csvfile.write(request)
+
+            os.system('python format_translations.py --validate --filename {f}'.format(f=fn))
+        if file_type.endswith('.folder'):
+            data = service.files().list(q="'" + file_id + "'" + " in parents").execute()
+            download_consent_files(data['files'], service)
+    os.system('python consent_form_json.py')
```

Comment:
As discussed previously, please try to avoid unnecessary `os.system` calls. This is a python script and it should be possible to run it directly as python without starting a separate process.

```
import other_module
other_module.run(*args, **kwargs)
```